client/js/screens/About/About.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import {Text} from 'react-native';
+
+const About = () => {
```

Comment:
For best practice, please accept `props` as an argument for your presentation components.

Reply:
completed.